### PR TITLE
Backend

### LINKED ISSUE #7: Parse rss feed

### PR DESCRIPTION
**Pull request by Edmond120**
Implement route('/videos')
which returns rss feed data containing all the channels (and their most recent videos) listed in a watchlist file

### CODE REVIEW

File: server/app/rss.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	return f'https://www.youtube.com/feeds/videos.xml?channel_id={channel_id}'
+
+def from_file(filename):
+	return feedparser.parse(filename)
+
+def from_channel_id(channel_id):
+	print(channel_id)
+	return feedparser.parse(get_rss_url(channel_id))
+
+def summery(watchlist_path):
```
Comment (aziz512) [inline server/app/rss.py:14]: summary

File: server/app/rss.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+def get_rss_url(channel_id):
+	return f'https://www.youtube.com/feeds/videos.xml?channel_id={channel_id}'
+
+def from_file(filename):
+	return feedparser.parse(filename)
+
+def from_channel_id(channel_id):
+	print(channel_id)
+	return feedparser.parse(get_rss_url(channel_id))
```
Comment (aziz512) [inline server/app/rss.py:12]: We need to get this data into a cleaned up json object. Currently, this gives a very noisy object with a lot of duplicate information and inconsistent naming patterns (title vs yt_videoid).  We should return a list of json objects with a handful of clear and relevant properties.

image

**Comment by aziz512** [comment]
We probably need very basic docs for the api methods

**Comment by Edmond120** [comment]
I'll move this to /videos_raw and create a simpler format for /videos
#10

**Pull request closed without merging**